src/rf/apps/workers/process.py

```diff
@@ -77,67 +78,36 @@ def _validate_image(self, data):
         processed.
         data -- attribute data from SQS.
         """
-        def mark_image_valid(s3_uuid):
-            image = LayerImage.objects.get(s3_uuid=s3_uuid)
-            image.status = enums.STATUS_VALID
-            image.save()
-            layer_id = image.layer_id
-
-            data = {'s3_uuid': s3_uuid}
-            self.queue.add_message(JOB_REPROJECT, data)
-
-            # Add a message to the queue that we can use to watch for timeouts.
-            data = {
-                'timeout': time.time() + TIMEOUT_SECONDS,
-                'layer_id': layer_id
-            }
-            self.queue.add_message(JOB_TIMEOUT, data, TIMEOUT_DELAY)
-            return True
-
-        def mark_image_invalid(s3_uuid, error_message):
-            image = LayerImage.objects.get(s3_uuid=s3_uuid)
-            image.status = enums.STATUS_INVALID
-            image.error = error_message
-            image.save()
-            layer_id = image.layer_id
-            layer = Layer.objects.get(id=layer_id)
-            layer.error = ERROR_MESSAGE_LAYER_IMAGE_INVALID
-            layer.status = enums.STATUS_FAILED
-            layer.status_updated_at = datetime.now()
-            layer.save()
-            return True
-
-        key = data['s3']['object']['key']
-        s3_uuid = self._extract_uuid_from_aws_key(key)
-        byte_range = '0-1000000'  # Get first Mb(ish) of bytes.
-
-        # Image validator thros AttributeError if it cannot read the image.
         try:
-            if ensure_band_count(key, byte_range):
-                return mark_image_valid(s3_uuid)
-            else:
-                error_message = ERROR_MESSAGE_IMAGE_TOO_FEW_BANDS
-                return mark_image_invalid(s3_uuid, error_message)
-        except AttributeError:
-            error_message = ERROR_MESSAGE_IMAGE_NOT_VALID
-            return mark_image_invalid(s3_uuid, error_message)
-
-    def _reproject(self, data):
-        """
-        Reproject incoming image to Web Mercator.
-        data -- attribute data from SQS.
-        """
-        # Pass image to Gdal to reproject
-        # If it succeeds continue else return False.
+            key = data['s3']['object']['key']
+        except KeyError:
+            return False
 
-        s3_uuid = data['s3_uuid']
-        image = LayerImage.objects.get(s3_uuid=s3_uuid)
-        image.status = enums.STATUS_REPROJECTED
-        image.save()
+        s3_uuid = self._extract_uuid_from_aws_key(key)
+        byte_range = '0-1000000'  # Get first Mb(ish) of bytes.
```

Comment:
We have an issue to deal with validating images that require full downloads as you saw earlier today. I think this is worth leaving here for now. It might be handy when we get to that issue.